takwimu/renderers.py

```diff
@@ -23,8 +23,17 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
 class JPEGRenderer(BaseRenderer):
     media_type = 'image/jpeg'
     format = 'jpg'
-    charset = None
     render_style = 'binary'
 
     def render(self, data, media_type=None, renderer_context=None):
         return data
+
+
+class FlourishHTMLRenderer(StaticHTMLRenderer):
```

Comment:
What are we adding to `StaticHTMLRenderer`?

Reply:
@kilemensi we're removing the extra checks it was doing before rendering, they were throwing unicode errors

Reply:
Cool @phillipahereza... Just add a comment at the top of the class there why we're adding it for future reference.

Reply:
@kilemensi done